app/assets/stylesheets/components/_profile-section.scss

```diff
@@ -13,6 +13,25 @@
 
 @media #{$breakpoint-sm} {
   .profile-info-box {
+    border-radius: $border-radius-md;
+    margin-bottom: $space-3;
+  }
+}
+
+.events-info-box {
+  border: $border-width solid $border-color;
+  border-radius: 0;
+  margin-bottom: 0;
+  overflow: hidden;
+
+  .bg-lightest-blue img {
+    margin-top: -2px;
+    vertical-align: middle;
+  }
+}
+
+@media #{$breakpoint-sm} {
```

Comment:
While there's an argument for consistency since it existed previously, for my own curiosity I'm wondering between the `@include at-media` and this use of breakpoint variables, if one or the other is the "preferred" approach. I see USWDS tends to use `at-media` in their examples?

Reply:
Using only USWDS is the goal, so I've replaced these breakpoints with the USWDS equivalent `at-media` 🙂